app/models/case_log.rb

```diff
@@ -15,13 +15,33 @@ def validate_property_number_of_times_relet(record)
     end
   end
 
+  def validate_api_reasonable_preference(record)
```

Comment:
I suppose technically someone could modify their browser request/turn off the javascript that shows/hides this so probably worth including it in the method below as well?

Reply:
Sorry you've lost me, what need to be added to the method?

Reply:
Ahh I see what you mean, I'll have a mess around with it.